doc/custom_nonconformity_score_proof.ipynb

```diff
@@ -0,0 +1,407 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a3995772",
+   "metadata": {},
+   "source": [
+    "The purpose of this documentation is to generalize the coverage guarantees of the [Predictive inference with the jackknife+](https://arxiv.org/pdf/1905.02928.pdf) methods (jacknife+ and jacknife mimax) to a larger set of nonconformity scores.\n",
+    "\n",
+    "First, two function sets are definied with properties that allow to verify the same coverage properties. Then, the two theorems guaranteing coverage for the jackknife+ method and the jackknife minmax one are reviewed with these new sets of functions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1615ad39",
+   "metadata": {},
+   "source": [
+    "___\n",
+    "# Nonconformity score and predition function\n",
+    "In order to generalize the coverage guarantee to more nonconformity scores (not only the absolute residuals), let first define two functions with some properties. The nonconformity score $f$ defined on $E^{2}$ ($E \\subset \\mathbb{R}$) and the prediction function $g$ defined on $F \\times E$ ($F \\subset \\mathbb{R}$). Uppercase letters are used to stick to the paper notation.\n",
+    "\n",
+    "## Nonconformity score\n",
+    "From an observation and a prediction, the function $f$ outputs the conformity score:\n",
+    "$$\\begin{align}\n",
+    "f:E^{2} &\\to F\\\\\n",
+    "(Y, \\hat{Y}) &\\mapsto f(Y, \\hat{Y})\n",
+    "\\end{align}$$\n",
+    "\n",
+    "It verifies the following properties:\n",
+    "$$\\begin{align}\n",
+    "& \\forall Y_{l} < Y_{h}: f(Y_{l}, \\hat{Y}) < f(Y_{h}, \\hat{Y}) \\\\\n",
+    "& \\forall \\hat{Y}_{l} < \\hat{Y}_{h}: f(Y, \\hat{Y}_{l}) > f(Y, \\hat{Y}_{h}) \\\\\n",
+    "& \\forall i \\neq j: R_{i,j} = f(Y_{i}, \\hat{\\mu}_{-(i,j)}(X_{i}))\n",
+    "\\end{align}$$\n",
+    "\n",
+    "## Prediction function\n",
+    "From a conformity score and a prediction, the function $g$ ouputs a perturbed prediction:\n",
```

Comment:
I fear that perturbed might be misleading in this context

Reply:
Would you prefer without "perturbed"?